File: 2231-find-first-palindromic-string-in-the-array/find-first-palindromic-string-in-the-array.cpp
class Solution {
public:
    string firstPalindrome(vector<string>& words) {
        string ans="";
        for(int i=0;i<words.size();i++){
            if(isPalindrome(words[i])){
                ans=words[i];
                break;
            }
        }
        return ans;
    }
    bool isPalindrome(string s){
        string temp=s;
        reverse(temp.begin(),temp.end());
        return s==temp;
    }
};

File: 2231-find-first-palindromic-string-in-the-array/find-first-palindromic-string-in-the-array.java
class Solution {
    public String firstPalindrome(String[] words) {
        String ans="";
        for(int i=0;i<words.length;i++){
            if(isPalindrome(words[i])){
                ans=words[i];
                break;
            }
        }
        return ans;
    }
    public boolean isPalindrome(String s){
        StringBuilder sb=new StringBuilder(s);
        sb.reverse();
        return s.equals(sb.toString());
    }
}